quiz.py

```diff
@@ -1,29 +1,35 @@
 from simple_chalk import chalk
+from time import sleep
 
 def start_quiz(questions):
     score = 0
     for i in range(0, len(questions)):
-        print(questions[i]["question"]+"\n")
+        print('--------------------------------------------------------')
+        print(f'Q{i+1}) {questions[i]["question"]}\n'); sleep(0.3)
+        
         for j in range(1, 5):
-            print(str(j)+" "+questions[i]["option"+str(j)])
-            print()
-        try:
-            answer = int(input())
-        except ValueError:
-            print(chalk.cyan('Please enter a choice between 1 and 4\n'))
+            print(f"({j}) {questions[i]['option'+str(j)]:30}", end='\t'); sleep(0.2)
+            if not j%2: print()
+        
+        while True:
```

Comment:
I intentionally , skipped the question if the input is not in between 1-4 , there are no do-overs in a quiz.

Reply:
oh okay... I will remove that part